app/routes/admin/sales/fees.js

```diff
@@ -5,17 +5,12 @@ export default Route.extend({
     return this.get('l10n').t('Fees');
   },
 
-  model() {
-    return [{
-      event   : 'Sample Event',
-      tickets : 1,
-      fee     : 20,
-      revenue : 25
-    }, {
-      event   : 'Sample Event2',
-      tickets : 3,
-      fee     : 10,
-      revenue : 40
-    }];
+  async model() {
+    let filterOptions = [];
+
+    return this.store.query('admin-sales-fee', {
+      filter       : filterOptions,
+      'page[size]' : 10
```

Comment:
Don't limit the page size to 10, rather for now define it as 0 to fetch all the records as admin sales table does not has pagination yet. If you like, you can work on converting it to an ember model table after this PR is merged.